front50-web/src/main/groovy/com/netflix/spinnaker/front50/controllers/PipelineController.groovy

```diff
@@ -117,6 +117,19 @@ class PipelineController {
     pipelineDAO.bulkImport(pipelines)
   }
 
+  @PreAuthorize("@fiatPermissionEvaluator.storeWholePermission() and hasPermission(#application, 'APPLICATION', 'WRITE')")
+  @RequestMapping(value = '{application}/reorder', method = RequestMethod.POST)
```

Comment:
I might suggest a new controller for these new operations and an endpoint along the lines of:

```
/actions/pipelines/reorder
```

or

```
/actions/reorderPipelines
```

Per the above comment, the payload could be changed slightly to pass in the application in the body.